src/Components/Stats.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import styles from '../CSS/Stats.module.css';
+import { Col, Row } from 'antd';
+import globalStyles from '../CSS/globalStyles.module.css';
+import HiddenStats from './HiddenStats';
+
+function Stats() {
+  const [isHidden, setisHidden] = useState(false);
+
+  return (
+    <div id="statsOverlay" className={styles.stats}>
+      <div className={`${styles.statsContainer} ${globalStyles.normalP}`}>
+        <Row>
+          <Col className={styles.statItem}>
+            <h2 className={globalStyles.headerTextmed}>Recidivism</h2>
+            {isHidden ? <HiddenStats /> : null}
+            <p>
+              According to data from the U.S. Bureau of Justice Statistics
+              (BJS), 67.8% of released state prisoners were arrested for a new
+              crime within three years, and 76.7% were arrested within five
+              years (Durose, Cooper, & Snyder, 2014).
+            </p>
+            <p>
+              Reducing recidivism not only protects society at large, but also
+              improves the life quality of individual ex-prisoners.
+            </p>
+            <div className={styles.statsButtonContainer}>
+              <button
+                className="btn btn-secondary"
+                onClick={() => setisHidden(!isHidden)}
+              >
+                {isHidden ? 'Less' : 'More'} Statistics
```

Comment:
Nit: For this can we change it from a button to a link with a down arrow since it shows more info below and an up arrow when it closes that info?  The button just feels bulky in this section.

Reply:
Also maybe a pointer on hover here?